social-network/src/main/java/com/levi9/socialnetwork/Service/impl/EmailServiceImpl.java

```diff
@@ -74,4 +76,21 @@ public String registerEmail(String name, String link) throws IOException {
 	}
 
 
+    @Async
+    public void sendNotificationAboutEventAsync(Event event, User user) throws MailException, InterruptedException {
+        Thread.sleep(1000);
```

Comment:
What is the reason for this? Thread sleep is generally bad practice.

Reply:
I used it for testing, forgot to remove it.